app/react/Library/actions/specs/libraryActions.spec.js

```diff
@@ -157,6 +160,7 @@ describe('libraryActions', () => {
 
     afterEach(() => backend.restore());
 
+    //eslint-disable-next-line max-statements
```

Comment:
:(

Reply:
i do not think this rule has that much value here, and i do not know how to properly fix it, wrapping some "it(" on describes will fix the problem but this is obviously not why we have this rule, anyways, one of the variables was not needed to be declared in this scope, removing it solved the issue for this particular case. :)